clang/test/CIR/CodeGen/vbase.cpp

```diff
@@ -41,10 +40,10 @@ void ppp() { B b; }
 // LLVM: $_ZTS1A = comdat any
 
 // LLVM: @_ZTV1B = linkonce_odr global { [3 x ptr] } { [3 x ptr] [ptr inttoptr (i64 12 to ptr), ptr null, ptr @_ZTI1B] }
-// LLVM: @_ZTT1B = linkonce_odr global [1 x ptr] [ptr getelementptr inbounds ({ [3 x ptr] }, ptr @_ZTV1B, i32 0, i32 0, i32 3)]
+// LLVM: @_ZTT1B = linkonce_odr global [1 x ptr] [ptr getelementptr inbounds nuw (i8, ptr @_ZTV1B, i64 24)]
```

Comment:
https://godbolt.org/z/qYxqKWh9n

has extra `inrange` but no `nuw`, also no constant folding.

Reply:
I cannot find this one!

Reply:
I think the output didn't include mangled symbols

```
@VTT for B = linkonce_odr dso_local unnamed_addr constant [1 x ptr] [ptr getelementptr inbounds inrange(-24, 0) ({ [3 x ptr] }, ptr @vtable for B, i32 0, i32 0, i32 3)], comdat, align 8
```